app/routes/api/maintainer.ts

```diff
@@ -0,0 +1,56 @@
+import { ActionFunction, json } from "@remix-run/node";
+import { getUserByUid } from "~/database.server";
+import { getProjectsMaintainedByUser } from "~/projects.server";
+
+/**
+ * This API endpoint verifies that a user is listed as the maintainer of at
+ * least 1 project in OSH. It expects a `userId` in the payload and a valid
+ * bearer token.
+ */
+export const action: ActionFunction = async ({ request }) => {
+  // Check that we have a valid userId in the payload
+  const data = await request.json();
+  const { userId } = data;
+  if (typeof userId !== "string") {
+    return new Response(null, {
+      status: 500,
```

Comment:
**Nitpick (blocking):** This should be a status code 400 error. General rule, 4xx the client did something wrong. 5xx error something went wrong on the server.

Reply:
Good catch, fixed!